osversion/windowsbuilds.go

```diff
@@ -35,4 +35,7 @@ const (
 
 	// V20H2 corresponds to Windows Server 20H2 (semi-annual channel).
 	V20H2 = 19042
+	
+	// corresponds to Windows Server 2022 (ltsc2022), and Windows 10 ().
```

Comment:
Start the comment with V2022 to satisfy godoc/go tooling. Wondering if we should name this 21H2 though

Reply:
I think `V21H2` is reasonable given it's the version identifier used here: https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info

Reply:
Ok cool